modules/Microsoft.CDN/profiles/endpoints/origins/deploy.bicep

```diff
@@ -0,0 +1,55 @@
+@description('Required. Object containing endpoint origin properties.')
+param originsProperties array
```

Comment:
This should ideally not be an object - and least of all an array

Reply:
Mistype in the description but that param has now been removed and replaced with the params from the resource type